src/constants/colors.ts

```diff
@@ -1,139 +1,21 @@
-};
-
-export const neutral = {
-  neutral10: '#F8F8F9',
-  neutral25: '#EDEDF0',
-  neutral50: '#E2E3E7',
-  neutral75: '#D6D7DE',
-
-  neutral100: '#C3C5CF',
-  neutral200: '#A9ACBA',
-  neutral300: '#8F93A5',
-  neutral400: '#6F748B',
-  neutral500: '#555B76',
-  neutral600: '#424867',
-  neutral700: '#484E6C',
-  neutral800: '#282F52',
-  neutral900: '#1C2139',
-};
-
-export const alert = {
-  veggy25: '#E6F6EF',
-
-  veggy200: '#82D2AE',
-  veggy500: '#00A45A',
-  veggy600: '#00733F',
-  veggy900: '#004727',
-
-  danger25: '#FEF0F0',
-
-  danger200: '#FBB1B1',
-  danger500: '#F52D2D',
-  danger600: '#AC1F1F',
-  danger900: '#450D0D',
-
-  alerty25: '#FFF8CF',
-
-  alerty200: '#FFEE8E',
-  alerty500: '#FFDE25',
-  alerty600: '#C29F00',
-  alerty900: '#383108',
-};
-
-export const extended = {
-  yellow25: '#FFFAF0',
-  yellow75: '#FFEFD0',
-
-  yellow100: '#FFE3AF',
-  yellow200: '#FFD689',
-  yellow300: '#FFCB69',
-  yellow400: '#FFBF48',
-  yellow500: '#FFB428',
-  yellow600: '#D99922',
-  yellow700: '#B37E1C',
-  yellow800: '#8C6316',
-  yellow900: '#47320B',
-
-  blue25: '#F2F1FF',
-  blue75: '#D7D4FE',
-
-  blue100: '#B7B1FD',
-  blue200: '#9C94FC',
-  blue300: '#8177FC',
-  blue400: '#6659FB',
-  blue500: '#4B3CFA',
-  blue600: '#4033D5',
-  blue700: '#352AAF',
-  blue800: '#29218A',
-  blue900: '#191455',
-
-  persian25: '#EFF1F8',
-  persian75: '#CDD5EA',
-
-  persian100: '#ABB8DB',
-  persian200: '#899BCD',
-  persian300: '#526CB5',
-  persian400: '#304FA7',
-  persian500: '#1C3E9E',
-  persian600: '#183586',
-  persian700: '#142B6F',
-  persian800: '#0F2257',
-  persian900: '#0B193F',
-
-  pink25: '#FEF0F3',
-  pink75: '#FDD1DB',
-
-  pink100: '#FBB1C2',
-  pink200: '#FA8CA4',
-  pink300: '#F86C8C',
-  pink400: '#F74D73',
-  pink500: '#F52D5A',
-  pink600: '#DF2952',
-  pink700: '#AC1F3F',
-  pink800: '#621224',
-  pink900: '#2F0911',
-
-  navy25: '#F6F6F9',
-  navy75: '#E8E9F0',
-
-  navy100: '#D9DBE6',
-  navy200: '#C4C7D8',
-  navy300: '#A8ACC6',
-  navy400: '#7E83AA',
-  navy500: '#696F9C',
-  navy600: '#454D85',
-  navy700: '#293272',
-  navy800: '#141E64',
-  navy900: '#0D1340',
-
-  teal25: '#EDFBFA',
-  teal75: '#C7F3EF',
-
-  teal100: '#A1EBE3',
-  teal200: '#73E1D6',
-  teal300: '#4DD9CB',
-  teal400: '#00322D',
-  teal500: '#00C8B4',
-  teal600: '#00B9A7',
-  teal700: '#00A494',
-  teal800: '#007468',
-  teal900: '#00322D',
-};
-
-export const statusColors = {
-  error: alert.danger500,
-  valid: alert.veggy500,
+export type TTextHexColors = '#FFFFFF' | '#C5C6CD' | '#0D0A38' | '#1EC06A' | '#EE0505';
+export type THeadingHexColors = '#FFFFFF' | '#0D0A38';
+export type TLinkHexColors = '#FFFFFF' | '#0D0A38';
+
+export const colors = {
+  base: {
+    primary: '#00B9A7',
+    secondary: '#4A3EDE',
+  },
+  neutral: {
+    white: '#FFFFFF',
+    light: '#EAEAEE',
+    medium: '#C5C6CD',
+    dark: '#0D0A38',
+  },
+  semantic: {
+    success: '#1EC06A',
+    alert: '#FFB428',
+    error: '#EE0505',
+  },
```

Comment:
Can we match those naming with the one in the brand document?

https://marvelapp.com/project/4236000/screen/60617108

Reply:
```
colors.tealPrimary
colors.tealSecondary
colors.bluePrimary
colors.blueSecondary
colors.surfaceDark
colors.surfaceBackground
```

Reply:
I don't think we should follow the design spec to the letter just yet as it's not fully fleshed out. There is plenty of overlap and we'd be running into issues, e.g.:
-  is `#0D0A38` `surfaceDark` or `textPrimary`, or `onSurfaceDark`?
- is `#FFFFFF`  `onSecondary`, `onSurfaceDark`, or just `White`?
etc.

Reply:
I think at the end of the day the important is that we have a restricted set of colors to use and clear guidelones on how to apply them 🎨 &nbsp;💅🏻

The design team has seen our colours spec and hasn't complain that the name should match theirs 😌

I would wait until the *new design system* matures... it's still plenty to do and spec and the new guidelines hasn't even been applied on any web product yet.